components/FeedbackView/ButtonGroup.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { JSX } from 'react';
+
+interface Variant {
+  title: string;
+  subtitle: string;
+  id: string;
+}
+
+interface ButtonGroupProps {
+  variants: Variant[];
+  handleFeedbackSubmit: (feedback: string) => void;
+}
+
+/**
+ * Component that renders a group of buttons.
+ * @param {object} props - The component props
+ * @param {object[]} props.variants - The list of button variants to render
```

Comment:
refactor (blocking): use proper type

```suggestion
 * @param {Variant[]} props.variants - The list of button variants to render
```